pko/package/manifest.yaml

```diff
@@ -47,7 +48,7 @@ spec:
             replicas:
               description: number of replicas for the unleash deployment
               type: integer
-              default: 3
+              default: 1
```

Comment:
1 replica may trigger DVO alert

Reply:
btw `resources` section, will `limits.cpu` always default to `100m` even not specified?

Reply:
Right, 1 is to conservative here